AppLovin MAX Demo App - Kotlin/app/src/main/res/layout/native_custom_ad_view.xml

```diff
@@ -55,17 +55,26 @@
 
     <TextView
         android:id="@+id/advertiser_text_view"
-        android:layout_width="wrap_content"
-        android:layout_height="0dp"
+        android:layout_width="56dp"
```

Comment:
Most of the changes here make sense based on our conversation but I'm not sure why we need to switch `android:layout_width="wrap_content"` to `android:layout_width="56dp"`. Generally it's better to avoid hardcoding values like this so I'm curious if there's a reason for it here (same with the Java app)

Reply:
yeah, that I just tested it and it works. For some reason I was under the impression that I had to set a specific value so the stars did not get covered, but I see your point that hard coding is not the best programming style.